tianshou/data/batch.py

```diff
@@ -728,23 +529,7 @@ def stack_(self,
     def stack(batches: List[Union[dict, 'Batch']], axis: int = 0) -> 'Batch':
         """Stack a list of :class:`~tianshou.data.Batch` object into a single
         new batch. For keys that are not shared across all batches,
-        batches that do not have these keys will be padded by zeros. E.g.
-        ::
-
-            >>> a = Batch(a=np.zeros([4, 4]), common=Batch(c=np.zeros([4, 5])))
-            >>> b = Batch(b=np.zeros([4, 6]), common=Batch(c=np.zeros([4, 5])))
-            >>> c = Batch.stack([a, b])
-            >>> c.a.shape
-            (2, 4, 4)
-            >>> c.b.shape
-            (2, 4, 6)
-            >>> c.common.c.shape
-            (2, 4, 5)
-
-        .. note::
-
-            If there are keys that are not shared across all batches, ``stack``
-            with ``axis != 0`` is undefined, and will cause an exception.
+        batches that do not have these keys will be padded by zeros.
```

Comment:
I think this doc can maintain here instead of delete. Same of cat and empty

Reply:
batch.py is already too long. We'd better keep it as short as possible. Given that we have a tutorial for Batch, I assume most users will learn that tutorial. So no need to duplicate the doc twice.

Reply:
I tend to agree with @Trinkle23897 Tutorial are not meant to be used as documentation. BUT documentation is not meant to be used as tutorial. So, try to find a good compromise. Only the required information to understand what each methods do and how they work, but those information have to be present.

Reply:
Agree with @duburcqa. You can make the doc shorter but please don’t erase the examples at all. It is easier for users to look at instead of an extra click to the Batch tutorial.

Reply:
Fixed in 17d907a .